src/Common/perf/PerfRunner/project.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
```

Comment:
Do you expect this perfrunner to be used by anyone other then corefx? If so it should be moved to BuildTools. If not then we can treat it like the RemoteExecuterConsoleApp like @mellinoe pointed out, but then it should be converted to build like that as well.